### PR TITLE
STR-25--BE: Send and email to the "Client" when a student completes a module

### PR DESCRIPTION
**Pull request by mbendandour**
# Description
Sends an email to admin when module of a user is complete  
# Testing
For specific research id and module name, log in as Matt first, then go to the '{your base url}//sendCompleteModuleMailTest' route. Browser will be blank.
 Depending on your .env file, if you are using **log** then check your storage/log for the admin email. Else, check whatever designated email you put under RECEIVE_EMAIL
# Installation
'Are there any dependencies the reviewer needs to download?'
no

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 Route::get('/get_group/{user_id}','UserGroupController@getGroup');
 
-Route::get('/sendMailTest','MailController@sendMail');
+Route::get('/sendMailTest','UserCompletesModuleEmailController@sendMail');
```
Comment (liadGolan) [inline routes/web.php]: I think a separate route for testing emails makes more sense here

**Review by liadGolan — CHANGES_REQUESTED** [review]
Besides this small change it looks good

**Review by liadGolan — CHANGES_REQUESTED** [review]
Attempting to send a test email results in a laravel error

**Review by liadGolan — APPROVED** [review]
👍

**Review by nercesk — APPROVED** [review]

**Pull request merged**